### PR TITLE
Add Moovit

### PR DESCRIPTION
**Pull request by leonardo-dgs**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -10927,6 +10927,23 @@
         ]
     },
 
+    {
+        "name": "Moovit",
+        "url": "https://support.moovitapp.com/hc/requests/new",
+        "difficulty": "hard",
+        "notes": "Contact the customer support and request account deletion. For \"Feedback Type\", choose \"Personal Data/GDPR\".",
```
Comment (tupaschoal) [inline _data/sites.json:10934]: Do they check that you live in a GDPR-covered country? If they require proof, then we should move to `limited`, if not, we can keep it as hard.

Comment (leonardo-dgs) [reply _data/sites.json:10934]: I am waiting for them to reply and delete my account. Then I can confirm it.

Comment (tupaschoal) [reply _data/sites.json:10934]: Ok!

**Review by tupaschoal — COMMENTED** [review]
Is this ready for merge?

**Comment by leonardo-dgs** [comment]
> Is this ready for merge?

No, I am still waiting that they delete my account. I will let you know when it's ready to merge.

**Review by tupaschoal — COMMENTED** [review]
Ready for merge now, I guess?

**Comment by leonardo-dgs** [comment]
They deleted my Moovit account, but not my Moovit Editor account, and I asked them to delete it.

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**